paper/paper.md

```diff
@@ -145,21 +145,15 @@ _--documentation_: Instead of traversing the whole file, this feature annotates
 
 !!! Check the NeXus links! Use either nexusformat.org links, or reference the path in the actual(!) pynxtools repo noting that pynxtools/definitions is a github submodule and content (as well as content path) can change version to version.
 
-# NOMAD integration (schema, parser)
+# NOMAD integration
 
-While pynxtools works as a standalone tool using the command line, it can also be integrated directly into Research Data Management Systems (RDMS). Out of the box, the package functions as a plugin within the NOMAD platform, converting and parsing data from experiments. This enables experimental data in the NeXus format to be integrated into NOMAD's metadata model, making it searchable and interoperable with other data from theory and experiment. The plugin consists of several key components (so called entry points):
+While pynxtools works as a standalone tool using the command line, it can also be integrated directly into Research Data Management Systems (RDMS). Out of the box, the package functions as a plugin within the NOMAD platform, converting and parsing data from experiments. This enables data in the NeXus format to be integrated into NOMAD's metadata model, making it searchable and interoperable with other data from theory and experiment.
```

Comment:
maybe we stop after "platform, " with a full-stop and rewrite the converting and parsing sentence such that we mention that this parsing is realized via the "__DataConverter__" functionality. I still like the idea that we give each functionality at least one sentence.